### PR TITLE
Exclude failing markers

### PR DESCRIPTION
**Pull request by techalchemy**
- Dependencies with markers that don't match the current environment will now be skipped during `pipenv lock`

### CODE REVIEW

**Comment by techalchemy** [comment]
Apparently I snuck in a script update for invoke and a changelog update

File: pipenv/core.py [hunk 1]
```diff
@@ -103,7 +101,7 @@
     ):
         INSTALL_LABEL = '🎅   '
     else:
-        INSTALL_LABEL = '🐍   '
+        INSTALL_LABEL = '🝝   '
```
Comment (uranusjr) [inline pipenv/core.py]: wait

File: tests/integration/test_sync.py [hunk 2]
```diff
@@ -22,7 +22,7 @@ def test_sync_error_without_lockfile(PipenvInstance, pypi):
 @pytest.mark.lock
 def test_mirror_lock_sync(PipenvInstance, pypi):
     with temp_environ(), PipenvInstance(chdir=True) as p:
-        mirror_url = os.environ.pop('PIPENV_TEST_INDEX', "https://pypi.python.org/simple")
+        mirror_url = os.environ.pop('PIPENV_TEST_INDEX', "https://pypi.kennethreitz..org/simple")
```
Comment (uranusjr) [inline tests/integration/test_sync.py]: Extra dot?

Comment (techalchemy) [reply tests/integration/test_sync.py]: Oops

**Pull request merged**